### PR TITLE
URL link was added

### PR DESCRIPTION
**Pull request by miosbtkyt**

### CODE REVIEW

**Comment by tak0kada** [comment]
Thank you for adding the link!

Is it common to separate the link into 2 lines like
```md
[title
](url)
```
? if not, would you make it into 1 line? Also, I would appreciate it if you fix my typo ``implment``.

You might like to modify README.md again and then,
```sh
git add README.md
git commit --amend  #modify the last commit
```
Thanks!

**Comment by tak0kada** [comment]
I find a reference about modifying the pull request: https://backlog.com/ja/git-tutorial/pull-request/07/
I coulld have added the comment this way.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # Learning course Modern Statistics for Modern Biology @ statistical genetics lab
 
 The course is basically on R, but we try to implment algorithms in other computer languages
+
+## Resource
+[Modern Statistics for Modern Biology
```
Comment (tak0kada) [inline README.md]: Thank you for adding the link!

Is it common to separate the link into 2 lines like
```md
[title
](url)
```
? if not, would you make it into 1 line?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 # Learning course Modern Statistics for Modern Biology @ statistical genetics lab
 
 The course is basically on R, but we try to implment algorithms in other computer languages
```
Comment (tak0kada) [inline README.md]: Would you fix my typo?

**Comment by miosbtkyt** [comment]
Great, Thank you !
Instead of `--amend`, I used `-m` when `git commit `.

**Review by tak0kada — CHANGES_REQUESTED** [review]
You might like to modify README.md again and then,
```sh
git add README.md
git commit --amend  #modify the last commit
```
Thanks!

**Comment by tak0kada** [comment]
Will you also modify the comment to ``add link to README.md``? Thank you for so many requests.

File: README.md (hunk 1 above)
Comment (miosbtkyt) [reply README.md]: typo...

File: README.md (hunk 2 above)
Comment (miosbtkyt) [reply README.md]: Trying implementing 
http://www.tounou.net/wordpress/?p=226

**Comment by miosbtkyt** [comment]
I used `git commit -m "blablabla..."` otherwise, `git push` didn't work :(

**Comment by tak0kada** [comment]
Thank you!

**Pull request merged**